src/confseq/misc.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+
+def superMG_crossing_fraction(mart_fn, dist_fn, alpha, repeats):
+    exceeded = [None] * repeats
+    for i in range(repeats):
+        x = dist_fn()
+        mart = mart_fn(x)
+        exceeded[i] = True if any(mart > 1 / alpha) else False
+
+    return np.mean(exceeded)
```

Comment:
i suggest at least
```suggestion
def superMG_crossing_fraction(mart_fn, dist_fn, alpha, repeats):
    def replication():
        x = dist_fn()
        mart = mart_fn(x)
        return any(mart > 1 / alpha)
    return np.mean(replication() for _ in range(repeats)
```
it's not really that bad as a one-liner,
```python
def superMG_crossing_fraction(mart_fn, dist_fn, alpha, repeats):
    return np.mean(any(mart_fn(dist_fn()) > 1 / alpha) for _ in range(repeats)
```
but i leave it up to your taste. (i'm not sure if numpy will take the generator expression but if not you can use a list comprehension or just `sum(<generator expression>) / repeats`.)

Reply:
Updated in #12